multi-arch-builders/provisioning/ppc64le/variables.tf

```diff
@@ -0,0 +1,87 @@
+
+variable "ibmcloud_api_key" {
+    description = "Denotes the IBM Cloud API key to use"
+    default = ""
+}
+
+variable "ibmcloud_region" {
+    description = "Denotes which IBM Cloud region to connect to"
+    default     = "us-south"
+}
+
+#INSERTED FOR MULTI-ZONE REGION SUCH AS FRANKFURT
+
+variable "ibmcloud_zone" {
+    description = "Denotes which IBM Cloud zone to connect to - .i.e: eu-de-1 eu-de-2  us-south etc."
+    default     = "us-south"
+}
+
+# Got the ID from `ibmcloud resource service-instances --long field` command, refer GUID for the instance
+variable "power_instance_id" {
+    description = "Power Virtual Server instance ID associated with your IBM Cloud account (note that this is NOT the API key)"
+    default = "556eb201-32bf-4ae2-8ab5-dfd7bbe97789"
+}
+
+
+# The PowerVs cost are high, check the price before adding
+# more processors and memory. This number may change
+# due the PowerVs availability.
+
+variable "memory" {
+    description = "Amount of memory (GB) to be allocated to the VM"
+    default     = "50"
+}
+
+variable "processors" {
+    description = "Number of virtual processors to allocate to the VM"
+    default     = "15"
+}
+
+# The s922 model is the cheapest model
+variable "system_type" {
+    description = "Type of system on which the VM should be created - s922/e880/e980"
+    default     = "s922"
+}
+
+variable "proc_type" {
+    description = "Processor type for the LPAR - shared/dedicated"
+    default     = "capped"
```

Comment:
The description mentions `shared/dedicated` as options but we have set it to `capped`?